.github/workflows/feature-branch.yml

```diff
@@ -51,10 +58,20 @@ jobs:
       # We should switch to nofilter once it's fixed
       filter-mode: diff_context
       suggestions: true
+      runs-on: ${{ inputs.runs-on }}
 
   ci_codeowners:
     uses: cloudposse/github-actions-workflows/.github/workflows/ci-codeowners.yml@main
     with:
       is_fork: ${{ github.event.pull_request.head.repo.full_name != github.repository }}
+      runs-on: ${{ inputs.runs-on }}
     secrets:
-      github_access_token: ${{ secrets.github_access_token }}
\ No newline at end of file
+      github_access_token: ${{ secrets.github_access_token }}
```

Comment:
@goruha is `github_` a reserved word? is this allowed?

Reply:
`github_token` is reserved